modules/packages/zypper

```diff
@@ -40,18 +41,20 @@ zypper_options = ["--quiet", "-n"]
 
 NULLFILE = open(os.devnull, 'w')
 
-
-# Suse zypper "--oldpackage" option is only supported greater that 1.6.169
-import rpm
-from distutils.version import StrictVersion
-
-ZYPPER_SUPPORTS_OLDPACKAGE=False
-package_zypper = rpm.TransactionSet().dbMatch('name', "zypper").next()
-if StrictVersion(package_zypper['version']) >= StrictVersion("1.6.169"):
-    ZYPPER_SUPPORTS_OLDPACKAGE=True
-
-
 redirection_is_broken_cached = -1
+zypper_supports_oldpackage_cached = -1
+
+def zypper_supports_oldpackage():
+    # Suse zypper "--oldpackage" option is only supported greater that 1.6.169
+    global zypper_supports_oldpackage_cached
+    if zypper_supports_oldpackage_cached == -1:
```

Comment:
I think None would be more appropriate than -1, but it seems to follow the existing "style" so, ok.